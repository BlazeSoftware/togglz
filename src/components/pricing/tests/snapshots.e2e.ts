import snapIt from '../../../test/snap-it';

describe('pricing', () => {
  const snap = snapIt('pricing-overview');
  snap('renders free plan correctly', '<pricing-overview plan="starter"></pricing-overview>');
  snap('renders pro plan correctly', '<pricing-overview plan="pro"></pricing-overview>');
});
