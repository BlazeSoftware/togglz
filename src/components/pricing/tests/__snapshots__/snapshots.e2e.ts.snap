// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pricing renders free plan correctly 1`] = `"<pricing-overview plan=\\"starter\\" class=\\"hydrated\\"><div class=\\"o-container o-container--large\\"><div class=\\"o-grid o-grid--wrap o-grid--xsmall-full o-grid--small-full o-grid--medium-full o-grid--large-fit\\"><div class=\\"o-grid__cell o-grid__cell--width-50\\"><div class=\\"c-card\\"><header class=\\"c-card__header\\"><h2 class=\\"c-heading\\">Starter</h2><br><h3 class=\\"c-heading\\">Best for individuals</h3><div class=\\"u-super\\">$0/mo</div><blaze-badge class=\\"hydrated\\"><!----><span class=\\"c-badge c-badge--success\\">current plan</span></blaze-badge></header><div class=\\"c-card__body\\"><div class=\\"u-small\\">Includes:</div><ul class=\\"c-list c-list--unstyled\\"><li class=\\"o-feature c-list__item\\"><i aria-hidden=\\"\\" class=\\"o-icon u-text--quiet fa-fw far fa-check-circle\\"></i>10 feature toggles</li><li class=\\"o-feature c-list__item\\"><i aria-hidden=\\"\\" class=\\"o-icon u-text--quiet fa-fw far fa-check-circle\\"></i>2 environments</li><li class=\\"o-feature c-list__item\\"><i aria-hidden=\\"\\" class=\\"o-icon u-text--quiet fa-fw far fa-check-circle\\"></i>10,000 API requests per month</li></ul></div><footer class=\\"c-card__footer c-card__footer--block\\"><div class=\\"c-input-group\\"></div></footer></div></div><div class=\\"o-grid__cell o-grid__cell--width-50\\"><div class=\\"c-card\\"><header class=\\"c-card__header\\"><h2 class=\\"c-heading\\">Pro</h2><br><h3 class=\\"c-heading\\">More traffic, more environments</h3><div class=\\"u-super\\">$199/mo</div><blaze-badge class=\\"hydrated\\"><!----><span class=\\"c-badge c-badge--info\\">more power</span></blaze-badge></header><div class=\\"c-card__body\\"><div class=\\"u-small\\">Includes <span class=\\"u-text--loud\\">Starter</span>, plus:</div><ul class=\\"c-list c-list--unstyled\\"><li class=\\"o-feature c-list__item\\"><i aria-hidden=\\"\\" class=\\"o-icon u-text--quiet fa-fw far fa-check-circle\\"></i>Unlimited feature toggles</li><li class=\\"o-feature c-list__item\\"><i aria-hidden=\\"\\" class=\\"o-icon u-text--quiet fa-fw far fa-check-circle\\"></i>Unlimited environments</li><li class=\\"o-feature c-list__item\\"><i aria-hidden=\\"\\" class=\\"o-icon u-text--quiet fa-fw far fa-check-circle\\"></i>Unlimited API requests</li></ul></div><footer class=\\"c-card__footer c-card__footer--block\\"><div class=\\"c-input-group\\"><button class=\\"c-button c-button--block c-button--info\\">Upgrade</button></div></footer></div></div></div></div></pricing-overview>"`;

exports[`pricing renders pro plan correctly 1`] = `"<pricing-overview plan=\\"pro\\" class=\\"hydrated\\"><div class=\\"o-container o-container--large\\"><div class=\\"o-grid o-grid--wrap o-grid--xsmall-full o-grid--small-full o-grid--medium-full o-grid--large-fit\\"><div class=\\"o-grid__cell o-grid__cell--width-50\\"><div class=\\"c-card\\"><header class=\\"c-card__header\\"><h2 class=\\"c-heading\\">Starter</h2><br><h3 class=\\"c-heading\\">Best for individuals</h3><div class=\\"u-super\\">$0/mo</div><blaze-badge class=\\"hydrated\\"><!----><span class=\\"c-badge c-badge--brand\\">free</span></blaze-badge></header><div class=\\"c-card__body\\"><div class=\\"u-small\\">Includes:</div><ul class=\\"c-list c-list--unstyled\\"><li class=\\"o-feature c-list__item\\"><i aria-hidden=\\"\\" class=\\"o-icon u-text--quiet fa-fw far fa-check-circle\\"></i>10 feature toggles</li><li class=\\"o-feature c-list__item\\"><i aria-hidden=\\"\\" class=\\"o-icon u-text--quiet fa-fw far fa-check-circle\\"></i>2 environments</li><li class=\\"o-feature c-list__item\\"><i aria-hidden=\\"\\" class=\\"o-icon u-text--quiet fa-fw far fa-check-circle\\"></i>10,000 API requests per month</li></ul></div><footer class=\\"c-card__footer c-card__footer--block\\"><div class=\\"c-input-group\\"><button class=\\"c-button c-button--block\\">Downgrade</button></div></footer></div></div><div class=\\"o-grid__cell o-grid__cell--width-50\\"><div class=\\"c-card\\"><header class=\\"c-card__header\\"><h2 class=\\"c-heading\\">Pro</h2><br><h3 class=\\"c-heading\\">More traffic, more environments</h3><div class=\\"u-super\\">$199/mo</div><blaze-badge class=\\"hydrated\\"><!----><span class=\\"c-badge c-badge--success\\">current plan</span></blaze-badge></header><div class=\\"c-card__body\\"><div class=\\"u-small\\">Includes <span class=\\"u-text--loud\\">Starter</span>, plus:</div><ul class=\\"c-list c-list--unstyled\\"><li class=\\"o-feature c-list__item\\"><i aria-hidden=\\"\\" class=\\"o-icon u-text--quiet fa-fw far fa-check-circle\\"></i>Unlimited feature toggles</li><li class=\\"o-feature c-list__item\\"><i aria-hidden=\\"\\" class=\\"o-icon u-text--quiet fa-fw far fa-check-circle\\"></i>Unlimited environments</li><li class=\\"o-feature c-list__item\\"><i aria-hidden=\\"\\" class=\\"o-icon u-text--quiet fa-fw far fa-check-circle\\"></i>Unlimited API requests</li></ul></div><footer class=\\"c-card__footer c-card__footer--block\\"><div class=\\"c-input-group\\"></div></footer></div></div></div></div></pricing-overview>"`;
